index.js

```diff
@@ -45,6 +45,12 @@ metagen.formats.amd = files => `define([
         ${files.map((file, i) => `'${noExt(file)}': arguments[${i}]`).join(',\n        ')}
     }
 });`
+metagen.formats.es6 = files => `${
+    files.map(file => `import ${varName(file)} from './${noExt(file)}';`).join('\n')
```

Comment:
Looks fine to me, but @sadasant I see a semicolon added as part of the `import` template string, but it is not part of the README. If the output of es6 metagen has semicolons, then we should reflect that in the README.

Documentation and output should be as close and transparent as possible.